### PR TITLE
solv: add plaindir repo support

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

**Comment by mlschroe** [comment]
You should ignore .patch.rpm and .delta.rpm files, like in tools/repo2solv.c's read_plaindir_repo() implementation.

**Comment by lnussel** [comment]
updated

**Comment by lnussel** [comment]
so any blockers that prevent merging?

**Comment by Conan-Kudo** [comment]
Why do we want this?

**Comment by mlschroe** [comment]
Thanks! (Although this is just example code, of course)

File: examples/solv/repoinfo_type_plaindir.c [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  if ((dp = opendir(cinfo->path)) == 0)
+    {
+      perror(cinfo->path);
+      return -1;
+    }
+  while ((de = readdir(dp)) != 0)
+    {
+      if (de->d_name[0] == 0 || de->d_name[0] == '.')
+        continue;
+      if (!endswith(de->d_name, ".rpm") || endswith(de->d_name, ".delta.rpm") || endswith(de->d_name, ".patch.rpm"))
```
Comment (Conan-Kudo) [inline examples/solv/repoinfo_type_plaindir.c:61]: `*.drpm` is also a DeltaRPM extension...

Comment (mlschroe) [reply examples/solv/repoinfo_type_plaindir.c:61]: Files not ending in `.rpm` are already ignored.

**Pull request merged**